.gitlab-ci.yml

```diff
@@ -76,27 +73,25 @@ pytest:
   script:
     - python -V
     # Command to test our application
-    - pytest -v
+    - pytest -vvv
 
 
 # Build
 
 build:
   image: docker:19.03.12
   stage: build
-  needs: ["pytest"]
+#  needs: ["pytest"]
   services:
-    - docker:19.03.12-dind
+    - docker:dind
```

Comment:
Why aren't we using the dependency proxy here?

Reply:
I removed for some reason I'll check